tests/test_FLEbasis2D.py

```diff
@@ -0,0 +1,330 @@
+import os
+
+import numpy as np
+import pytest
+
+from aspire.basis import FBBasis2D, FLEBasis2D
+from aspire.image import Image
+from aspire.nufft import backend_available
+from aspire.numeric import fft
+from aspire.source import Simulation
+from aspire.utils import utest_tolerance
+from aspire.volume import Volume
+
+from ._basis_util import UniversalBasisMixin
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
+
+
+def show_fle_params(basis):
+    return f"{basis.nres}-{basis.epsilon}"
+
+
+def gpu_ci_skip():
+    pytest.skip("1e-7 precision for FLEBasis2D.evaluate()")
+
+
+def odd_resolution_skip():
+    pytest.skip("FB matching for odd resolution.")
+
+
+fle_params = [
+    (32, 1e-4),
+    (32, 1e-7),
+    (32, 1e-10),
+    (32, 1e-14),
+    (33, 1e-4),
+    (33, 1e-7),
+    (33, 1e-10),
+    (33, 1e-14),
+]
+
+test_bases = [
+    FLEBasis2D(L, epsilon=epsilon, dtype=np.float64, match_fb=False)
+    for L, epsilon in fle_params
+]
+
+test_bases_match_fb = [
+    FLEBasis2D(L, epsilon=epsilon, dtype=np.float64) for L, epsilon in fle_params
+]
+
+
+def create_images(L, n):
```

Comment:
I don't think there's a way to have this fixture interact with the `mark.parametrize` parameters without duplication because it parametrizes over the same param (that is, resolution). Or there may be but it's very complicated (from my browsing the pytest docs)

Reply:
It would have to be combined with the other fixture we decided to make an issue for.  No worries. Thanks.